visualization/tslint.json

```diff
@@ -10,6 +10,7 @@
         "new-parens": true,
         "no-unused-variable": [true, {"ignore-pattern": "^_"}],
         "member-access": true,
+        "member-ordering": [true, {"order": "instance-sandwich"}],
```

Comment:
remove public->protected->private ordering. Functions should be declared right after they're called